src/libs/ajax.js

```diff
@@ -890,6 +890,8 @@ const Workspaces = signal => ({
         return fetchRawls(`${root}/entities/delete`, _.mergeAll([authOpts(), jsonBody(entities), { signal, method: 'POST' }]))
       },
 
+      getEntities: entityType => fetchRawls(`${root}/entities/${entityType}`, _.merge(authOpts(), { signal })).then(r => r.json()),
```

Comment:
I haven't tried it with large tables. That's unfortunate if that endpoint doesn't work as advertised. Though since creating a version fetches entity metadata, this could easily be changed to use [entityQuery](https://rawls.dsde-prod.broadinstitute.org/#/entities/entityQuery).

Reply:
That's good that it can be switched over to entityQuery if necessary- I do think AJ wants to eventually remove the listEntitiesOfType endpoint entirely. The limit they added is pretty high, but I think some of the users we're targeting for the feature preview may have tables large enough to hit it.

Here's the relevant Rawls PR that added the limit: https://github.com/broadinstitute/rawls/pull/1647

Reply:
Switched endpoints in 4a16b32f6478d69ff5bb2fd2fdb6617f81c4b3c4.

Reply:
Nice! That commit LGTM, still a 👍